agent/src/main/java/org/openremote/agent/protocol/mqtt/AbstractMQTT_IOClient.java

```diff
@@ -123,8 +128,12 @@ protected AbstractMQTT_IOClient(String clientId, String host, int port, boolean
             .applyAutomaticReconnect();
 
         if (secure) {
-            builder = builder.sslWithDefaultConfig();
-        }
+			MqttClientSslConfigBuilder sslBuilder = MqttClientSslConfig.builder();
+			sslBuilder = sslBuilder.keyManagerFactory(keyManagerFactory);
```

Comment:
As mentioned below; it may be that only a custom trust store is needed, don't know what this method does if null is supplied